### DISCUSSION TITLE
Add a 'none' formatter

### DISCUSSION THREAD

**Discussion by gisostallenberg** [composer-unused/composer-unused#516] [Ideas]
I would like to have no output and just use the exit code in my CI.

Would you be interested in a 'none' formatter (or any other suggested name)?

**Comment by gisostallenberg** [comment]
In addition I would like to add a compact formatter, which only lists unused packages

**Comment by icanhazstring** [comment]
That makes sense.
Would do a `--no-output` option to suppress the output.

For the other one, what do you mean by "only as list"? Could you give an example?

**Reply by gisostallenberg** [reply]
> Would do a --no-output option to suppress the output.

I thought that a formatter would be better than adding another option. Already implemented in https://github.com/composer-unused/composer-unused/pull/517

> For the other one, what do you mean by "only as list"? Could you give an example?

```
Unused packages: symfony/console, symfony/string
```

**Reply by icanhazstring** [reply]
Not quite a fan of using an empty formatter just to get an empty output and exit code tho 🤔

The output is a default behavior. So it should be actively suppressed with an option.

Problem is, the exit code is calculated in the formatters, that is already a design flaw from my side 😉

**Comment by llaville** [comment]
Additonal `none` formatter is not necessary. If you specify `--quiet` option on command line. 

To run and get status code returned : try 

```
composer-unused --quiet
echo $?
```
Will return either 0 (OK), 1(KO)

PS: Works with latest version 0.8.8

NOTE: `--quiet` is default option on standard Symfony Console Application

**Reply by gisostallenberg** [reply]
👍🏽 

Are you interested in the 'compact' formatter? Then I'll change the PR, otherwise feel free to close it.

**Reply by icanhazstring** [reply]
Thanks @llaville 
@gisostallenberg yes compact is fine 👍
Maybe also add single line for zombies and ignored ones as well.
